src/index.js

```diff
@@ -3,8 +3,28 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
+import firebase from '@firebase/app';
+import '@firebase/firestore';
+import { FirestoreProvider } from 'react-firestore';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+const config = {
+  apiKey: 'AIzaSyDIYdDfqZAJ_BXyxtZL1gHBf5brDjOVQJA',
+  authDomain: 'tcl-8-smart-shopping-lis-a6069.firebaseapp.com',
+  databaseURL: 'https://tcl-8-smart-shopping-lis-a6069.firebaseio.com',
+  projectId: 'tcl-8-smart-shopping-lis-a6069',
+  storageBucket: 'tcl-8-smart-shopping-lis-a6069.appspot.com',
+  messagingSenderId: '471591173463',
+  appId: '1:471591173463:web:2981efcaef0dfe01939212',
+};
+
+firebase.initializeApp(config);
```

Comment:
You'll find this same code already save in `src/lib/firebase.js`. You can keep it in either place, but let's make sure it's not duplicated.